tests/robot-tests/tests/general_public/methodology_page_absence.robot

```diff
@@ -73,7 +73,7 @@ Search for "pupil"
     user verifies accordion is closed    1. Overview of absence statistics
 
     user enters text into element    id:pageSearchForm-input    pupil
-    user waits until element contains    id:pageSearchForm-resultsLabel    Found 127 results
+    user waits until element contains    id:pageSearchForm-resultsLabel    Found 128 results
```

Comment:
This is because the seed test data contains sections where the user has created their own Contact Us section within the user-defined data. This search box appears to just do an html search rather than  (as well as?)  a database call, so adding one more occurrence of the Contact Us data yields one more result. 

(The word "pupil" is present in the title of the publication, which is displayed in the Contact Us section)